app/templates/components/cron-job.hbs

```diff
@@ -1,6 +1,10 @@
 <div class="cron-job-value branch-name">{{cron.branch.name}}</div>
 <div class="cron-job-value interval">{{cron.interval}}</div>
-<div class="cron-job-value next-enqueuing">{{pretty-date cron.next_enqueuing}}</div>
+{{#if isOverdue}}
+  <div class="cron-job-text next-enqueuing">{{"soon"}}</div>
```

Comment:
If we're just returning a string here, no need for the `{{ }}` syntax. We can just do:

```
<div class="cron-job-text next-enqueuing">soon</div>
```